### ISSUE TITLE
Unable to update sslvserver configuration

### ISSUE THREAD

**Issue by rbjoergensen** [citrix/terraform-provider-citrixadc#164]
Are there any plans to add the options in this endpoint so you can configure the SSL parameters on an sslvserver.
https://developer-docs.citrix.com/projects/netscaler-nitro-api/en/11.1/configuration/ssl/sslvserver/sslvserver/

If I try to bind an SSL profile I get the following error.
```
│ Error: [ERROR] go-nitro: Failed to create resource of type sslvserver_sslciphersuite_binding, name=csvs_test_https, 
err=failed: 599 Netscaler specific error ({ "errorcode": 3740, "message": "Operation not permitted. To do this Disable 
default ssl profile by setting 'set ssl parameter -defaultProfile D'", "severity": "ERROR" })
│
│   on resources.tf line 30, in resource "citrixadc_csvserver" "example_csvs_https":
│   30: resource "citrixadc_csvserver" "example_csvs_https" {
```

The config I'm using looks like this.
``` tf
resource "citrixadc_csvserver" "example_csvs_https" {
  name          = "csvs_test_https"
  ipv46         = "192.168.1.51"
  port          = 443
  servicetype   = "SSL"
  sslcertkey    = "nginx.domain.local"
  sslprofile    = citrixadc_sslprofile.csvs_sslprofile.name
  casesensitive = "OFF"
}

resource "citrixadc_sslprofile" "csvs_sslprofile" {
    name  = "sp_test"
    hsts  = "DISABLED"
    ssl3  = "DISABLED"
    tls1  = "DISABLED"
    tls11 = "DISABLED"
    tls12 = "ENABLED"
    tls13 = "ENABLED"
    ecccurvebindings = ["ALL"]

    cipherbindings {
      ciphername     = "DEFAULT"
      cipherpriority = 1
    }
}
```
I've tried with defaultprofile D and E and I get the same error.

**Comment by giorgos-nikolopoulos** [comment]
It seems you try to bind a ciphersuite directly to the vserver.
The error you are seeing is from NITRO prompting you to disable the default ssl profile to allow this operation.

This operation should be done from the ADC nscli. In general it is a one time configuration.

The configuration you have included in your message does not show the source of the error.

There should be a lbvserver or csvserver with the `ciphersuites` parameter set to a value.

As far as I know you cannot disable the default ssl profile once it is enabled.
The way to bind ciphersuites to ssl vservers with the default profile enabled is through the sslprofile that binds to the sslvserver as you show in your sample configuration.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #217
